src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -147,6 +170,20 @@ public MesosCloud(
     }
   }
 
+  /**
+   * Copy constructor.
+   * Allows to create copies of the original mesos cloud. Since it's a singleton
+   * by design, this method also allows specifying a new name.
+   * @param name Name of the cloud to be created
+   * @param source Source Mesos cloud implementation
+   * @since TODO
```

Comment:
see above. "since" Javadoc field requires a version, but it can be specified only when/if the PR gets merged